package.json

```diff
@@ -11,7 +11,8 @@
       "import": "./dist/autoconsent.extra.esm.js",
       "require": "./dist/autoconsent.extra.cjs.js"
     },
-    "./rules/rules.json": "./rules/rules.json"
+    "./rules/rules.json": "./rules/rules.json",
+    "./rules/consentomatic.json": "./rules/consentomatic.json"
```

Comment:
Could you update the readme docs about this change in exports? It could be a breaking change for downstream users relying on rules.json